.editorconfig

```diff
@@ -571,6 +571,17 @@ MA0053.exceptions_should_be_sealed = true
 # Also report diagnostic for types that define (new) virtual members.
 MA0053.class_with_virtual_member_shoud_be_sealed = true
 
+# MA0076: Do not use implicit culture-sensitive ToString in interpolated strings
+#
+# Interpolated strings have performance benefits.
+dotnet_diagnostic.MA0076.severity = none
+
+# MA0089: Optimize string method usage
+#
+# Duplicate of CA1865-CA1867 /
```

Comment:
Can you give me a few examples?

Reply:
I think these are new analyzers in net8.0 sdk, but I couldn't get them to fire (maybe they are aware of the other tfms which don't have the replacement methods?).

I could only get CA1847

![image](https://github.com/sshnet/SSH.NET/assets/5132141/11985b6b-36be-48e0-8f6b-eef983870252)

and some other similar looking warnings

![image](https://github.com/sshnet/SSH.NET/assets/5132141/50ebaa8e-6395-4e93-a77c-2d6c5c9831b5)

I can revert, pending discussion on string.Join